src/EnergyPlus/UnitarySystem.cc

```diff
@@ -14924,6 +14937,7 @@ namespace UnitarySystems {
         this->m_HeatingPartLoadFrac = PartLoadFrac;
         this->m_HeatingSpeedRatio = SpeedRatio;
         this->m_HeatingCycRatio = CycRatio;
+        HeatCoilLoad = ReqOutput;
```

Comment:
What if the coil didn't meet ReqOutput? Looks like the other coils are doing the same thing.